mayland-blocks/style.css

```diff
@@ -12,7 +12,7 @@ License: GNU General Public License v2 or later
 License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
 Template: blockbase
 Text Domain: mayland-blocks
-Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, block-patterns, rtl-language-support, theme-options, threaded-comments, translation-ready, wide-blocks
+Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, wpcom-fse, block-patterns, rtl-language-support, theme-options, threaded-comments, translation-ready, wide-blocks
```

Comment:
I think this theme should keep the `full-site-editing` tag?

Reply:
Yes it should, thanks.